### ISSUE TITLE
Cannot load pre-trained model

### ISSUE THREAD

**Issue by WLatonov** [Jhonve/GCN-Denoiser#14]
An error occurrs when I try to load pre-trained model. The error is catched after the line

GCN_1 = torch::jit::load(firstModelPath);

is executed.

**Comment by therateful** [comment]
have you solved the problem?

**Comment by WLatonov** [comment]
> have you solved the problem?

Yes. You need to have the same GPU type when you train model and when you load model.

**Comment by therateful** [comment]
what's the meaning of "GPU type"?Could you please give me an example?

**Comment by WLatonov** [comment]
NVIDIA Tesla V100, or something else.
